Helm Release Notes for 4.11 (#47720)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])

> * There is currently a known issue when a user is creating an instance of ProjectHelmChartRepository custom resource (CR) or HelmChartRepository CR with TLS verification. As a result, the user cannot list the repositories and perform Helm-related operations with the ProjectHelmChartRepository and HelmChartRepository that supports TLS verification. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])

Reviewer: Is this only for ProjectHelmChartRepository or also for HelmChartRepository @Kartikey-star

> * There is currently a known issue when a user is creating an instance of ProjectHelmChartRepository custom resource (CR) or HelmChartRepository CR with TLS verification. As a result, the user cannot list the repositories and perform Helm-related operations with the ProjectHelmChartRepository and HelmChartRepository that supports TLS verification. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])

Author: Done.

* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])

> * There is currently a known issue when a user is creating an instance of ProjectHelmChartRepository custom resource (CR) with TLS verification. As a result, the user cannot list the repositories and perform Helm-related operations with the ProjectHelmChartRepository that supports TLS verification. There is currently no workaround for this issue.(link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])

Reviewer 2: ```suggestion
* There is currently a known issue. If you create an instance of `ProjectHelmChartRepository` custom resource (CR) that uses TLS verification, you cannot list repositories and perform Helm-related operations. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
```

Reviewer 2: Please consider making these changes.

Author: Done.
